packages/core/src/Services/socket-general.js

```diff
@@ -235,7 +235,7 @@ const BinarySocketGeneral = (() => {
                 if (active_platform === 'DBot') return;
 
                 client_store.logout().then(() => {
-                    let redirect_to = routes.trade;
+                    let redirect_to = routes.traders_hub;
```

Comment:
this is logging out logic from which place @sergei-deriv ?

Reply:
@mitra-deriv It's error handler if we have errors in WS response